llvm/include/llvm/MC/MCELFObjectWriter.h

```diff
@@ -78,6 +78,8 @@ class MCELFObjectTargetWriter : public MCObjectTargetWriter {
         return ELF::ELFOSABI_FREEBSD;
       case Triple::Solaris:
         return ELF::ELFOSABI_SOLARIS;
+      case Triple::OpenBSD:
```

Comment:
This should be separate with a test https://reviews.llvm.org/D107748

Reply:
Thanks @MaskRay, #98158 has just that, with such tests